funcobjtask/fntask1.py

```diff
@@ -0,0 +1,10 @@
+def func(arg1, arg2, arg3, *args, arg4=5, **argsdict):
+    test_var_1 = 100
+    test_var_2 = "Test variable"
+    test_var_3 = 500
+    test_var_4 = "Some text"
+    test_var_5 = 1
+    return len(locals()) - len([var for var in locals() if var.startswith("arg")])  # excluding the arguments of function
```

Comment:
Why do we exclude arguments? They are also local variables (they become local variables when you call a function)

Reply:
It was mentioned in task that you need to to detect the number of local variables which are declared in a function.

Reply:
So I thought only about variables that are declared in function's body